README.md

```diff
@@ -1,10 +1,147 @@
-# steps-sign-apk
+# Android Sign ![Bitrise Build Status](https://app.bitrise.io/app/3b968e65d584db2a.svg?token=Yk1LUEjLZtIjeIW4OOZvKw&branch=master) [![Bitrise Step Version](https://shields.io/github/v/release/bitrise-steplib/steps-sign-apk?include_prereleases)](https://github.com/bitrise-steplib/steps-sign-apk/releases) ![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg) [![Bitrise Community](https://img.shields.io/badge/community-Bitrise%20Discuss-lightgrey)](https://discuss.bitrise.io/)
 
-## How to use this Step
+Signs your APK or Android App Bundle to be uploaded to the Google Play Store.
 
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
+Check Bitrise for a [detailed description](https://www.bitrise.io/integrations/steps/sign-apk).
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+## Examples
+
+### Build a bundle and sign it
+
+```yml
+---
```

Comment:
I'm not sure we should include this whole example. Or at least we should somehow highlight the sign apk part